escape_roomba/event_logger.py

```diff
@@ -6,43 +6,40 @@
 
 from escape_roomba.format_util import fid, fobj
 
+logger = logging.getLogger('bot.events')
```

Comment:
In the other classes you used a private `_logger`, but not here. Seems inconsistent, unless you plan other modules to use this events logger as well?